services/port_manager/src/main/java/com/futurewei/alcor/portmanager/util/SpringContextUtil.java

```diff
@@ -13,7 +13,7 @@
         See the License for the specific language governing permissions and
         limitations under the License.
 */
-package com.futurewei.alcor.common.utils;
+package com.futurewei.alcor.portmanager.util;
```

Comment:
Should we keep them in common?

Reply:
I am not familiar with port manager ~ we could ask Piaoping if we should move them into commom

Reply:
oh I thought you moved this file to portmanager. If you only change the reference, it is okay then.